pkg/scalers/redis_scaler.go

```diff
@@ -30,6 +31,7 @@ type redisMetadata struct {
 	listName         string
 	address          string
 	password         string
+	dbIdx            int
```

Comment:
Let's rename this to `databaseIndex` maybe?
```suggestion
	databaseIndex            int
```

Will be easier for new folks to understand what it represents imo